plugins/parcelport/mpi/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ if(HPX_WITH_PARCELPORT_MPI)
     #    hpx_add_link_flag_if_available(${MPI_CXX_LINK_FLAGS})
   endif()
 
+  if(MPI_CXX_COMPILE_DEFINITIONS)
+    add_compile_definitions(${MPI_CXX_COMPILE_DEFINITIONS})
```

Comment:
I tried this initially. As it turned out, `add_parcelport` does not create targets for static parcelports. Adding the definitions directly to the `hpx` target is also impossible as it does not exist yet. Since the parcelports need to be created before generating the `hpx``target, using a global setting is currently the only option to propagate the settings.

Since we plan to overhaul the parcelport build system anyway, this is only a (really ugly) stop-gap measure.